pkg/engine/armvariables_test.go

```diff
@@ -234,30 +229,6 @@ func TestK8sVars(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	expectedMap["cloudInitFiles"] = map[string]interface{}{
```

Comment:
We don't need to redefine the `cloudInitFiles` map for this 18.04 non-VHD-specific test, as it's the same as the preceding 16.04 non-VHD test